src/providers/auth/context.js

```diff
@@ -14,7 +14,7 @@ const AuthContext = createContext(null);
 
 export function AuthProvider({ children }) {
   const [reqHeader, setReqHeader] = useState({
-    "Device-id": browserName,
+    "Device-id": browserName + 3234242,

```

Comment:
What is this value?

Reply:
oww sorry that will change when merge, for testing purpose only